src/Components/FirmwareSelector/index.jsx

```diff
@@ -269,7 +266,7 @@ function FirmwareSelector({
         <div className="gui-box grey">
           <div className="gui-box-titlebar">
             <div className="spacer-box-title">
-              {t('selectTarget')}
+              {`${t('selectTarget')}${esc ? ` (${esc.displayName})` : ''}`}
```

Comment:
esc check can be omitted if we have a proper default set in the prop validation.